website/versioned_docs/version-22.4/MockFunctionAPI.md

```diff
@@ -0,0 +1,310 @@
+---
```

Comment:
Can only add this to the master docs? Will ship in the 23.1 version

Reply:
Ah, I misunderstood the process. Yeah, np - will remove my `versioned_docs` changes